internal/presentation/presentation.go

```diff
@@ -575,7 +575,10 @@ func (rk resultKey) selectVarValue(result rego.Result) (interface{}, bool) {
 		return result.Bindings[rk.varName], true
 	}
 	val := result.Expressions[rk.exprIndex].Value
-	if _, ok := val.(bool); ok {
+	if f, ok := val.(bool); ok {
+		if f {
+			return true, true
+		}
```

Comment:
I think we can actually simplify this function:

1. If `varName` is non-empty, return the binding value (above).
2. Otherwise, return `val`.

This way the function is simpler--it always generates a value for each result key. It will prioritize variable bindings over expression values which is what we want anyway.

Reply:
This works properly! Thanks.
And after this change, the second return value `bool` is not needed,
because it will always become `true`.
So I removed the lines and tests passed as before.